examples/basic-examples/modules/ces/ces.tf

```diff
@@ -5,16 +5,16 @@
 #}
 
 resource "opentelekomcloud_ces_alarmrule" "alarm_rule" {
-  alarm_name        = "${var.alarm_name}"
-  alarm_description = "${var.alarm_desc}"
+  alarm_name        = var.alarm_name
+  alarm_description = var.alarm_desc
   metric {
     #"namespace" = "SYS.ECS"
     namespace   = "SYS.AS"
     metric_name = "as"
     dimensions {
       name = "AutoScalingGroup"
-      #"value" = "${opentelekomcloud_compute_instance_v2.webserver.id}"
-      value = "${var.as_group_id}"
+      #"value" = opentelekomcloud_compute_instance_v2.webserver.id
```

Comment:
should we remove this comment?

Reply:
I was not removing comments, examples still to be checked